app/api/v1/models/meetups.py

```diff
@@ -1,17 +1,29 @@
 from datetime import datetime
 
-class Meetups:
+Meetups = []
+class Meetup:
     """ Meetups constructor """
```

Comment:
Nice and neat. However, it's inappropriate to have two initializers in the same class

Reply:
Thanks will check on that